lib/hydra/pcdm/models/file.rb

```diff
@@ -5,6 +5,16 @@ class File < ActiveFedora::File
     metadata do
       configure type: RDFVocabularies::PCDMTerms.File
       property :label, predicate: ::RDF::RDFS.label
+
+      property :file_name, predicate: EBUCoreVocabularies::EBUCoreTerms.filename 
+      property :file_size, predicate: EBUCoreVocabularies::EBUCoreTerms.fileSize 
+      property :date_created, predicate: EBUCoreVocabularies::EBUCoreTerms.dateCreated
+      property :has_mime_type, predicate: EBUCoreVocabularies::EBUCoreTerms.hasMimeType
+      property :date_modified, predicate: EBUCoreVocabularies::EBUCoreTerms.dateModified
+      property :byte_order, predicate: SweetjplVocabularies::SweetjplTerms.byteOrder
+
+      # This is a server-managed predicate which means Fedora does not let us change it.
+      property :file_hash, predicate: RDF::Vocab::PREMIS.hasMessageDigest
```

Comment:
Also, I've opened PRs to use premis:hasMessageDigest for the SHA1 digest, and to make the mimeType and filename modifiable:

https://github.com/fcrepo4/fcrepo4/pull/790
https://github.com/fcrepo4/fcrepo4/pull/791

We'll talk about this at today's tech metadata call: https://wiki.duraspace.org/display/hydra/Technical+Metadata+Call+2015-05-12